### PR TITLE
Bump Traefik to v1.3.0

### PR DESCRIPTION
**Pull request by ldez**
Hello,

This PR updates traefik to 1.3.0.

/cc @vdemeester @emilevauge

Cheers

### CODE REVIEW

File: library/traefik [hunk 1]
```diff
@@ -1,19 +1,11 @@
...
-
-Tags: v1.2.3, 1.2.3, v1.2, 1.2, morbier, latest
-GitCommit: 50d631ddf197da6228172cb86ab6098d875a6ea0
-
-Tags: v1.2.3-alpine, 1.2.3-alpine, v1.2-alpine, 1.2-alpine, morbier-alpine
-GitCommit: 50d631ddf197da6228172cb86ab6098d875a6ea0
-Directory: alpine
+Tags: v1.3.0-alptine, 1.3.0-alptine, v1.3-alptine, 1.3-alptine, raclette-alptine
+GitCommit: 9ccd85a426ec167ad8c5e441f91fdff1790461b7
+Directory: alptine
```
Comment (tianon) [inline library/traefik]: "alptine" 😀

Comment (vdemeester) [inline library/traefik]: 😂

Comment (ldez) [inline library/traefik]: sorry, fixed 🙃

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 0efdce8..d31fbad 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,21 +1,11 @@
-traefik:1.2
-traefik:1.2-alpine
-traefik:1.2.3
-traefik:1.2.3-alpine
 traefik:1.3
 traefik:1.3-alpine
-traefik:1.3.0-rc3
-traefik:1.3.0-rc3-alpine
+traefik:1.3.0
+traefik:1.3.0-alpine
 traefik:latest
-traefik:morbier
-traefik:morbier-alpine
 traefik:raclette
 traefik:raclette-alpine
-traefik:v1.2
-traefik:v1.2-alpine
-traefik:v1.2.3
-traefik:v1.2.3-alpine
 traefik:v1.3
 traefik:v1.3-alpine
-traefik:v1.3.0-rc3
-traefik:v1.3.0-rc3-alpine
+traefik:v1.3.0
+traefik:v1.3.0-alpine
diff --git a/traefik_latest/Dockerfile b/traefik_latest/Dockerfile
index 4468614..22d7458 100644
--- a/traefik_latest/Dockerfile
+++ b/traefik_latest/Dockerfile
@@ -9,5 +9,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \    
-      org.label-schema.version="v1.2.3" \
+      org.label-schema.version="v1.3.0" \
       org.label-schema.docker.schema-version="1.0"
\ No newline at end of file
diff --git a/traefik_latest/traefik b/traefik_latest/traefik
index d528ebe..e07e946 100755
Binary files a/traefik_latest/traefik and b/traefik_latest/traefik differ
diff --git a/traefik_morbier-alpine/Dockerfile b/traefik_morbier-alpine/Dockerfile
deleted file mode 100644
index 31271d9..0000000
diff --git a/traefik_morbier-alpine/Dockerfile.tmpl b/traefik_morbier-alpine/Dockerfile.tmpl
deleted file mode 100644
index a34e53b..0000000
diff --git a/traefik_morbier-alpine/entrypoint.sh b/traefik_morbier-alpine/entrypoint.sh
deleted file mode 100755
index 49b3a32..0000000
diff --git a/traefik_morbier-alpine/traefik b/traefik_morbier-alpine/traefik
deleted file mode 100755
index d528ebe..0000000
diff --git a/traefik_raclette-alpine/Dockerfile b/traefik_raclette-alpine/Dockerfile
index b28f067..5e25918 100644
--- a/traefik_raclette-alpine/Dockerfile
+++ b/traefik_raclette-alpine/Dockerfile
@@ -13,5 +13,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \    
-      org.label-schema.version="v1.3.0-rc3" \
+      org.label-schema.version="v1.3.0" \
       org.label-schema.docker.schema-version="1.0"
\ No newline at end of file
diff --git a/traefik_raclette-alpine/traefik b/traefik_raclette-alpine/traefik
index 169e4ef..e07e946 100755
Binary files a/traefik_raclette-alpine/traefik and b/traefik_raclette-alpine/traefik differ
diff --git a/traefik_raclette/Dockerfile b/traefik_raclette/Dockerfile
deleted file mode 100644
index 0a03b0f..0000000
diff --git a/traefik_raclette/Dockerfile.tmpl b/traefik_raclette/Dockerfile.tmpl
deleted file mode 100644
index 8fd822c..0000000
diff --git a/traefik_raclette/certs/ca-certificates.crt b/traefik_raclette/certs/ca-certificates.crt
deleted file mode 100644
index 0455528..0000000
diff --git a/traefik_raclette/traefik b/traefik_raclette/traefik
deleted file mode 100755
index 169e4ef..0000000
```

</details>

**Comment by yosifkit** [comment]
Build test of #2998; 13923d87 (`traefik`):

```console

$ bashbrew build traefik:v1.3.0
Building bashbrew/cache:713a7a71872cedd83218bb5de3935f0853179f79968134cb88345b861ee2014f (traefik:v1.3.0)
Tagging traefik:v1.3.0
Tagging traefik:1.3.0
Tagging traefik:v1.3
Tagging traefik:1.3
Tagging traefik:raclette
Tagging traefik:latest

$ test/run.sh traefik:v1.3.0
testing traefik:v1.3.0
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.3.0-alpine
Building bashbrew/cache:ffa079ee2bf6677756ce89c6ac378d3568aa50c47f1806448fa571673c14b345 (traefik:v1.3.0-alpine)
Tagging traefik:v1.3.0-alpine
Tagging traefik:1.3.0-alpine
Tagging traefik:v1.3-alpine
Tagging traefik:1.3-alpine
Tagging traefik:raclette-alpine

$ test/run.sh traefik:v1.3.0-alpine
testing traefik:v1.3.0-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...passed

```

**Pull request merged**